packages/mwp-app-render/src/util/dateTimeUtils.test.js

```diff
@@ -0,0 +1,44 @@
+import {
+        padNumber,
+        getFormattedTime,
+        convertToLocalTime,
+} from './dateTimeUtils';
+
+const timeUTC = 1234567890; // fixed millisecond value
+const eventOffset = -1234567; // fixed millisecond value
+const runtimeDate = new Date(timeUTC + eventOffset); // time value determines whether local offset is in daylight savings
+const runtimeOffset = runtimeDate.getTimezoneOffset() * 60 * 1000; // runtime-dependent millisecond offset
+
+describe('padNumber', () => {
+	it('should pad numbers less than 10', () => {
+		for (let i = 0; i < 10; i += 1) {
+			expect(padNumber(i)).toBe(`0${i}`);
+		}
+	});
+	it('should not pad numbers greater or equal to 10', () => {
+		for (let i = 10; i < 60; i += 1) {
```

Comment:
I think the range of 10-60 wouldn't be as meaningful as `10, 100, 1000, 10000` because then you're testing the string formatting of longer strings - I can't think of reasonable implementations that would handle 11 differently than 12, but I can definitely think of implementations that would fail to account for 3-digit numbers even if it handled 2-digit numbers beautifully

```js
for (let i = 10; i < 10001; i *= 10) {
```

or a little more explicit:

```js
for (pow = 1; pow < 5; pow += 1) {
  const test = Math.pow(10, pow);
  expect(padNumber(test).toBe(test.toString())
}
```